app/mailers/archive_devise_mailer.rb

```diff
@@ -27,4 +27,16 @@ def confirmation_instructions(record, token, opts = {})
     subject = t("users.mailer.confirmation_instructions.subject", app_name: ArchiveConfig.APP_SHORT_NAME)
     devise_mail(record, :confirmation_instructions, opts.merge(subject: subject))
   end
+
+  def password_change(record, opts = {})
+    @pac_footer = true
```

Comment:
Do we want the PAC footer on the admin email? I don't know if we should be telling admins to contact PAC if they're getting unexpected password change emails.